tests/test_helpers/dom_helper.ts

```diff
@@ -195,14 +195,20 @@ function dispatchEvent(selector: string | EventTarget, ev: Event) {
   }
 }
 
-export function setInputValueAndTrigger(
+export async function setInputValueAndTrigger(
   selector: DOMTarget,
   value: string,
-  eventType: string
-): void {
+  mode?: "onlyInput" | "onlyChange"
```

Comment:
what do you think of two different functions: `changeInputValue` and `inputValue` ? would it be clearer than "onlyInput" | "onlyChange" ?